stdlib/Test/src/Test.jl

```diff
@@ -60,7 +60,7 @@ function scrub_backtrace(bt)
 end
 
 function scrub_exc_stack(stack)
-    return Any[ (x[1], scrub_backtrace(x[2])) for x in stack ]
+    return Any[ (x[1], scrub_backtrace(x[2]::Vector{Union{Ptr{Nothing},Base.InterpreterIP}})) for x in stack ]
```

Comment:
```suggestion
    return Any[ (x[1], scrub_backtrace(x[2]::Vector{Union{Ptr{Nothing},Base.InterpreterIP,Core.Compiler.InterpreterIP}})) for x in stack ]
```
I'm not sure if this is at all necessary, but this can happen if you collect your stacktrace from Core.Compiler.Timing (used by SnoopCompile to record data about entrances to inference). Such stacktraces seem unlikely to get passed to Test, so I'd leave this to your judgement.

Reply:
I followed code a bit and found this backtrace is generated from [`Base.catch_stack` here](https://github.com/JuliaLang/julia/blob/d524f21917031ab6acc91f1531021596748e5c12/stdlib/Test/src/Test.jl#L523), and so I'd like to leave this it as is.